code/modules/reagents/Chemistry-Reagents.dm

```diff
@@ -82,8 +84,9 @@
 	removed = M.get_metabolism(removed)
 	max_dose = max(volume, max_dose)
 
-	if(overdose && (dose > overdose) && (location != CHEM_TOUCH))
-		overdose(M, alien, removed, dose/overdose)
+	if(overdose && (volume > overdose) && (dose > od_minimum_dose) && (location != CHEM_TOUCH)) //OD based on volume in blood, but waits for a small amount of the drug to metabolise before kicking in.
+		is_overdosed = TRUE
```

Comment:
Not really a fan of this variable being kept, it looks to me like you can just check dosage, and that's a better way anyway because as it is this is never cleared, so even if you go under the OD threshold it'll stay TRUE

Reply:
Sorted.